R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2253,6 +2253,15 @@ test_that("mutate(), transform(), rename() and names()", {
   detach(airquality)
 })
 
+test_that("alias on SparkDataFrame", {
+  df <- alias(read.df(jsonPath, "json"), "table")
```

Comment:
instead of adding a new test, add to one already naming things to reuse an existing df?